src/internals/strings/Strings.ts

```diff
@@ -435,6 +435,29 @@ export namespace Strings {
     return: H.KebabCase<this["arg0"]>;
   }
 
+  interface ToRecordFn extends Fn {
+    return: this["args"] extends [infer union extends string, infer value]
+      ? Record<union, value>
+      : never;
+  }
```

Comment:
I feel like this would make more sense under `Union`. It's similar to Join, that takes a tuple of strings but is under `Tuples`.

I don't know how this would work but it would be nice if we could use O.Create and arg0 as a key to build a record.

Reply:
Or maybe move it to `Objects.Record` since this has the same API as the builtin `Record` generic